oshi-core/src/main/java/oshi/hardware/common/AbstractCentralProcessor.java

```diff
@@ -87,9 +87,9 @@ protected AbstractCentralProcessor() {
             Set<Integer> pkgCoreKeys = this.logicalProcessors.stream()
                     .map(p -> (p.getPhysicalPackageNumber() << 16) + p.getPhysicalProcessorNumber())
                     .collect(Collectors.toSet());
-            List<PhysicalProcessor> physProcs = pkgCoreKeys.stream().sorted()
-                    .map(k -> new PhysicalProcessor(k >> 16, k & 0xffff)).collect(Collectors.toList());
-            this.physicalProcessors = Collections.unmodifiableList(physProcs);
+            this.physicalProcessors = pkgCoreKeys.stream().parallel().sorted()
+                .map(k -> new PhysicalProcessor(k >> 16, k & 0xffff))
```

Comment:
Not needed, sorted gets everything back into a single thread where order matters.